public/main.js

```diff
@@ -57,11 +60,17 @@ token = getToken();
 let loginButtonStatus = () => {
     let checkToken = localStorage.getItem("token");
     let logoutButton = document.querySelector('.logout-button');
+    let savedRecipes = document.querySelector('.recipes-container')
     if (checkToken === null) {
         logoutButton.textContent = 'Log In';
+        document.querySelector('.view-saved')
+          .classList.add('hidden');
     }
     else if (checkToken !== null) {
         logoutButton.textContent = 'Log Out';
+        savedRecipes.removeChild(savedRecipes.firstChild);
```

Comment:
I believe this may be the line you were referencing in your notes? If so, I think you will want a while loop similar to lines 92-94. I also noticed that the recipes didn't clear when clicking log out, so I created a branch to address this, but I didn't see this pull request until after pushing mine up. If you update your code to a while loop to address the issue, then we can just ignore my pull request.